### PR TITLE
[Bugfix] Settings responsiveness

### PR DESCRIPTION
**Pull request by Civolilah**
Here are screenshots of the Settings page on different screen sizes:

Lg:

Screenshot 2023-01-18 at 19 23 10

Md:

Screenshot 2023-01-18 at 19 23 22

@turbo124 In my opinion, this is the clearest solution for our case. If we want everything to fit on the md screen we will have to get smaller text and other things and it won't look good. Let me know how this looks for you now.

### CODE REVIEW

**Comment by turbo124** [comment]
@Civolilah 

the issue here is the dropdowns  don't do anything onChange.

File: src/components/layouts/Settings.tsx [hunk 1]
```diff
@@ -163,24 +165,19 @@ export function Settings(props: Props) {
...
+            withBlank
+          >
             {basic.map((item) => (
-              <option
-                key={item.name}
-                value={item.href}
-                className={classNames(
-                  item.current
-                    ? 'bg-gray-200 text-gray-900'
-                    : 'text-gray-600 hover:bg-gray-50 hover:text-gray-900',
```
Comment (Civolilah) [inline src/components/layouts/Settings.tsx:179]: Those props have no effect on the option HTML element. That's the reason for the removal.

**Comment by Civolilah** [comment]
> 

@turbo124 The functionality part is also fixed.

**Review by beganovich — APPROVED** [review]
Looks good, thanks :+1:

**Pull request merged**